src/App.css

```diff
@@ -1,8 +1,21 @@
-.App {
+.BangWords {
   text-align: center;
+  background-color: #454d5e;
+  min-height: 100vh;
+}
+
+.BangWords-header {
+  background-color: #282c34;
+  min-height: 15vh;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(10px + 2vmin);
+  color: white;
 }
 
-.App-logo {
+/* .App-logo {
```

Comment:
I'm not 100p, but I think we can delete all of this stuff if it's leftover from the default App template

Reply:
Oh yeah, lol. Good catch. Also, the colors I just took from the template as well. I was thinking in a final version we could do a western 🤠